mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -37,7 +37,7 @@ class DistanceSensorItem {
 		send_tf(false),
 		sensor_id(0),
 		field_of_view(0),
-		orientation(-1),
+		orientation("LOCAL_NED"),
```

Comment:
Also, this is supposed to be https://github.com/mavlink/mavros/pull/342/files#diff-2adf7423447a0e5f4880b56161efdda9R51, so how can I init as an integer, use it as a string (which it is)?!